### PR TITLE
update readme and code

### PR DESCRIPTION
**Pull request by elenapeskova**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,125 +1,86 @@
...
-* **[Default.aspx](./CS/Default.aspx) (VB: [Default.aspx](./VB/Default.aspx))**
-* [Default.aspx.cs](./CS/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/Default.aspx.vb))
-<!-- default file list end -->
-# ASPxGridView - Batch Editing - A simple implementation of an EditItemTemplate
+# Grid View for ASP.NET Web Forms - How to implement an edit item template in batch mode
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/t115096/)**
 <!-- run online end -->
 
+This example demonstrates how to create an edit item template, add an editor to the template, and specify the grid's cell edit functionality in batch mode.
```
Comment (dmitry-eliseev-devexpress) [inline Readme.md]: > **specify** the grid's cell edit **functionality**

I do not understand how one can "specify some functionality". What did you actually mean by that?

File: Readme.md [hunk 2]
```diff
@@ -1,125 +1,86 @@
...
+
+    ```aspx
+    <dx:GridViewDataColumn FieldName="C1">
+        <EditItemTemplate>
+            <dx:ASPxSpinEdit ID="C1_spinEdit" runat="server" ClientInstanceName="C1spinEdit" Width="100%" />
+        </EditItemTemplate>
+    </dx:GridViewDataColumn>
+    ```
+
+2. Handle the grid's client-side [BatchEditStartEditing](https://docs.devexpress.devx/AspNet/js-ASPxClientGridView.BatchEditStartEditing) event. In the handler, use the [rowValues](https://docs.devexpress.devx/AspNet/js-ASPxClientGridViewBatchEditStartEditingEventArgs.rowValues) argument property to get the value of the processed cell, call the editor's `SetValue` method to assign the cell value to the editor, and focus the editor.
```
Comment (dmitry-eliseev-devexpress) [inline Readme.md]: Look, you list three actions that one should do in the handler, and the first list item is quite long. Such lists are quite difficult to read and understand. It's much better to make an explicit bulleted list instead. Like this:

```suggestion
2. Handle the grid's client-side [BatchEditStartEditing](https://docs.devexpress.devx/AspNet/js-ASPxClientGridView.BatchEditStartEditing) event and do the following in the handler:

   * Use the [rowValues](https://docs.devexpress.devx/AspNet/js-ASPxClientGridViewBatchEditStartEditingEventArgs.rowValues) argument property to get the value of the processed cell.
   * Call the editor's `SetValue` method to assign the cell value to the editor.
   * Focus the editor.
```

P.S. I hope we can use nested lists in GitHub.

File: Readme.md [hunk 3]
```diff
@@ -1,125 +1,86 @@
...
-
-```
-
-
-<p> </p>
-<p>5. Finally, handle the editor's client-side <a href="https://documentation.devexpress.com/AspNet/DevExpressWebASPxEditorsScriptsASPxClientTextEdit_KeyDowntopic.aspx">KeyDown</a> and <a href="https://documentation.devexpress.com/AspNet/DevExpressWebASPxEditorsScriptsASPxClientEdit_LostFocustopic.aspx">LostFocus</a> events to emulate the behavior of standard grid editors when an end-user uses a keyboard or mouse:</p>
+5. Handle the editor's client-side `KeyDown` and `LostFocus` events to emulate the editor behavior when a user presses a key or clicks outside the editor.
 
+> **Note**  
+> When you implement an edit item template, the control does not update the data source automatically. Handle the grid's server-side [RowUpdating](https://docs.devexpress.devx/AspNet/DevExpress.Web.ASPxGridView.RowUpdating), [RowInserting](https://docs.devexpress.devx/AspNet/DevExpress.Web.ASPxGridView.RowInserting), and [RowDeleting](https://docs.devexpress.devx/AspNet/DevExpress.Web.ASPxGridView.RowDeleting) events to manually update the data source.
```
Comment (dmitry-eliseev-devexpress) [inline Readme.md]: > to **manually** update the data source.

Word order. "manually" is a complement that describes circumstances and answers the question "how?" The complement should follow the main sentence part, not break it in two. You did it right first when said

> the control does not update the data source **automatically**.

```suggestion
> When you implement an edit item template, the control does not update the data source automatically. Handle the grid's server-side [RowUpdating](https://docs.devexpress.devx/AspNet/DevExpress.Web.ASPxGridView.RowUpdating), [RowInserting](https://docs.devexpress.devx/AspNet/DevExpress.Web.ASPxGridView.RowInserting), and [RowDeleting](https://docs.devexpress.devx/AspNet/DevExpress.Web.ASPxGridView.RowDeleting) events to update the data source manually.
```

**Review by dmitry-eliseev-devexpress — APPROVED** [review]
**Structure and Semantics**

This is a simple step-by-step instruction with code examples. The topic serves its purpose well but you need to rewrite the very first sentence because it's unclear what you've meant.

**Images**

The image is clear and complies with DevExpress image guidelines. Well done!

**Language**

In general, your language lives up to the highest DevExpress writing standards, and I commend you for the effort. I found only one issue related to word order. Please draw your attention towards this scenario to avoid such issues in the future.

One more thing -- not directly related to language but to writing in general -- every time you enumerate multiple items (three or more) in a sentence, consider using a bulleted list -- long lists in a row are not that easy to read and process. This issue becomes more immediate when you have long and complex items that could have potentially been separate sentences (like your first enumerated item in my corresponding comment).

**Bottom Line**

This is a good topic, and I can approve it right away. However, we need to clarify the first sentence before we can move further. Thanks in advance.

File: Readme.md (hunk 1 above)
Comment (elenapeskova) [reply Readme.md]: Here I mean the following: if we implement an edit item template, we should handle BatchEditStartEditing, BatchEditEndEditing, and (if necessary) BatchEditRowValidating events to aplly custom edit rules.

Comment (dmitry-eliseev-devexpress) [reply Readme.md]: It's impossible to "specify the functionality". You can configure it, customize it but definitely not specify. Do you agree with the following change?

```suggestion
This example demonstrates how to create an edit item template, add an editor to the template, and configure the grid's cell edit functionality in batch mode.
```

File: CS/web.config [hunk 4]
```diff
@@ -1,52 +1,61 @@
...
+      <section name="themes" type="DevExpress.Web.ThemesConfigurationSection, DevExpress.Web.v22.1, Version=22.1.3.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a" requirePermission="false" />
+      <section name="compression" type="DevExpress.Web.CompressionConfigurationSection, DevExpress.Web.v22.1, Version=22.1.3.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a" requirePermission="false" />
+      <section name="settings" type="DevExpress.Web.SettingsConfigurationSection, DevExpress.Web.v22.1, Version=22.1.3.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a" requirePermission="false" />
+      <section name="errors" type="DevExpress.Web.ErrorsConfigurationSection, DevExpress.Web.v22.1, Version=22.1.3.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a" requirePermission="false" />
+      <section name="resources" type="DevExpress.Web.ResourcesConfigurationSection, DevExpress.Web.v22.1, Version=22.1.3.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a" requirePermission="false" />
     </sectionGroup>
   </configSections>
+  <!--
+    For a description of web.config changes see http://go.microsoft.com/fwlink/?LinkId=235367.
```
Comment (albertov05) [inline CS/web.config]:
```suggestion
    For a description of web.config changes, see http://go.microsoft.com/fwlink/?LinkId=235367.
```
